streams/service/src/test/resources/custom-udf-microtest/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- Does not need a <parent> declaration -->
+
+    <groupId>com.hortonworks.streamline.test</groupId>
+    <artifactId>custom-udf-microtest</artifactId>
+    <version>0.1.0</version> <!--fixed version - update manually -->
```

Comment:
Okay, that would be one thing it could usefully inherit from a parent :-)
I'll see if the <minimizeJar> shading command prevents it from pulling in all the parent's dependencies, and if so go back to letting it have streamline pom as parent.